packages/static-wado-webserver/lib/middlewares/index.mjs

```diff
@@ -0,0 +1,9 @@
+import setCorsMiddlewares from "./cors.mjs";
+
+/**
+ * Set all app middlewares
+ * @param {*} app
+ */
+export default function setMiddlewares(app) {
```

Comment:
Should have the configuration object so it can decide on whether or not to include cors, and if so, with what backends allowed.